ci/validate-vendor.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+# Copyright 2017 The Go Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+#
+# This script checks if we changed anything with regard to dependency management
+# for our repo and makes sure that it was done in a valid way.
+#
+# This file is a slightly modified copy of https://github.com/golang/dep/blob/master/hack/validate-vendor.bash
+# The only change made was checking if `dep` was available and installing it if necessary.
```

Comment:
Isn't this check done with `dep-install` in `Makefile`?

Reply:
Yup, just deleted it